App.js

```diff
@@ -1,21 +1,40 @@
-import { StatusBar } from 'expo-status-bar';
 import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import { NavigationContainer } from '@react-navigation/native';
+import { createStackNavigator } from '@react-navigation/stack';
+import { LandingPage } from './src/LandingPage/LandingPage';
+import { NavBar } from './src/NavBar/NavBar';
+// import { Form } from './src/Form/Form';
+// import { Map } from './src/Map/Map';
+// import { ActiveTrip } from './src/ActiveTrip/ActiveTrip';
+
+
+const Stack = createStackNavigator();
 
 export default function App() {
   return (
-    <View style={styles.container}>
-      <Text>Open up App.js to start working on your app!</Text>
-      <StatusBar style="auto" />
-    </View>
+    <NavigationContainer>
+      <Stack.Navigator initialRouteName='Landing'>
```

Comment:
This stack navigator looks promising. I'm interested to learn more about it, to see if it is going to suit our needs!

Reply:
From what I've read and from other peoples experience when it comes to react native and routing this is the best/ most intuitive way to go with it. Basically, it's just set up on the app.js and is the only place that the routing needs to be from my understanding.